.github/workflows/ci.yml

```diff
@@ -49,20 +44,14 @@ jobs:
       - name: Checkout branch
         uses: actions/checkout@v5.0.0
 
-      - name: Set up Python
-        uses: actions/setup-python@v6.0.0
-        with:
-          python-version: "3.12.1"
-
-      - name: Set up KLC Check
+      - name: Set up KiCad Library Utils
         # FIXME: Temporarily pinned to v8 to pass tests
```

Comment:
Move this FIXME comment to right above the `version: "v8"` line. Do it for both CI jobs.